modules/account-lib/src/coin/eth/contractCall.ts

```diff
@@ -4,9 +4,9 @@ import { addHexPrefix, toBuffer } from 'ethereumjs-util';
 export class ContractCall {
   private _methodId: string;
   private _types: string[];
-  private _params: string[];
+  private _params: any[];
```

Comment:
What is the impact we are perceiving by opening up the _params to be a broad stroke change from string?

Reply:
Having this type be string also prevents passing of arrays of types, and this change was necessary to support batch transfers of ERC1155s

Reply:
@ppong this entire library is deprecated: https://github.com/ethereumjs/ethereumjs-abi

Can we avoid continuing to write new code that depends on it

Reply:
@tylerlevine I agree we should probably not use `ethereumjs` - however at the same time I think we should use the `ContractCall` class. I think we should migrate from `ethereumjs` to `ethers.js` in a separate ticket.

Reply:
I'm fine with that, could you please create the ticket though and leave it in a TODO comment in the source code?